ansible/roles_ocp_workloads/ocp4_workload_fraud_detection_usecase/templates/starburst/cache/cache-deploymentconfig.yml.j2

```diff
@@ -0,0 +1,104 @@
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: 'true'
+  name: postgresql
+  generation: 2
+  namespace: {{ ocp4_workload.starburst.namespace }}
+  labels:
+    template: postgresql-ephemeral-template
+    template.openshift.io/template-instance-owner: 2303c49b-2d7b-4389-9485-7e1f0d5f1540
```

Comment:
are these labels required for the Deployment config?

Reply:
This is remnant from the [ocp4_workload_starburst](https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_starburst/templates/postgres-deploymentconfig.j2) configuration. I can test with the label removed.